python/tvm/relay/transform/quantize/_quantize_pass.py

```diff
@@ -0,0 +1,82 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Relay pass wrapping the quantization and calibration workflow."""
+
+from typing import List
+
+import tvm
+from tvm.relay.transform.quantize import Quantizer, Calibrater, Requantizer, QuantizerPattern
+from .. import function_pass
+
+
+@function_pass(opt_level=5)
+class QuantizePass:
+    """Explicit relay pass wrapper around quantization workflow.
+
+    Parameters
+    ----------
+    quantizer_pattern_list : List[QuantizerPattern]
+        The patterns we want to quantize.
+
+    params : dict of str to NDArray
+        Parameters you would pass into relay.build or relay.build_module. We need params
+        so that we can run parts of the graph during calibration.
+
+    skip_first : bool
+        If True, we do not quantize the first quantizable pattern in the function. If False,
+        we will quantize it.
+
+    skip_last : bool
+        If True, we do not quantize the last quantizable pattern in the function. If False,
+        we will quantize it."""
+
+    def __init__(
+        self,
+        quantizer_pattern_list: List[QuantizerPattern],
```

Comment:
I wonder if it would be good to have a set of utilities to create default pattern lists with various calibration methods.